test/util/util.go

```diff
@@ -598,3 +606,51 @@ func Expect1PodOfEach(ctx context.Context, client *kubernetes.Clientset, ns stri
 	}, timeout).Should(gomega.BeEmpty())
 	ginkgo.By(fmt.Sprintf("Waited for ready pods in namespace %q with name prefixes %v", ns, namePrefixes))
 }
+
+func ScaleDeployment(ctx context.Context, client *kubernetes.Clientset, ns string, name string, target int32) {
+	ginkgo.By(fmt.Sprintf("Scaling Deployment %q in namespace %q to %d", name, ns, target))
+	gomega.EventuallyWithOffset(1, func() error {
+		gotSc, err := client.AppsV1().Deployments(ns).GetScale(ctx, name, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		gotSc.Spec.Replicas = target
+		_, err = client.AppsV1().Deployments(ns).UpdateScale(ctx, name, gotSc, metav1.UpdateOptions{})
+		if err != nil {
+			return err
+		}
+		gotDeploy, err := client.AppsV1().Deployments(ns).Get(ctx, name, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		labelSelector := metav1.FormatLabelSelector(gotDeploy.Spec.Selector)
+		podList, err := client.CoreV1().Pods(ns).List(ctx, metav1.ListOptions{
+			LabelSelector: labelSelector,
+		})
+		if err != nil {
+			return err
+		}
+		if len(podList.Items) != int(target) {
+			return fmt.Errorf("pod count not converged yet")
+		}
+		if gotDeploy.Status.ReadyReplicas != target {
```

Comment:
Why `ReadyReplicas` instead of `AvailableReplicas` ?

Reply:
Used `AvailableReplicas` instead.